Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -867,7 +867,7 @@ private class TabManagerDataSource: NSObject, UICollectionViewDataSource {
         if !tab.displayTitle.isEmpty {
             tabCell.accessibilityLabel = tab.displayTitle
         } else {
-            tabCell.accessibilityLabel = AboutUtils.getAboutComponent(tab.url)
+            tabCell.accessibilityLabel = tab.url?.getAboutComponent() ?? "" // Not sure why we do this, makes no sense I think.
```

Comment:
I agree this doesn't make sense. Let's just remove it rather than adding a comment about it?